dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1456,6 +1456,18 @@
       "properties": {
         "stringIndexType": {
           "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "responseFormat": {
+          "type": "string",
+          "description": "The format in which the response to be returned. If simple is selected we only return the entities result, if detailed is included we include the metadata.",
+          "enum": [
+            "simple",
+            "detailed"
+          ],
+          "x-ms-enum": {
+            "name": "ResponseFormat",
+            "modelAsString": true
```

Comment:
I recommend also providing documentation for these values. See https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-enum, specifically `values`.

Reply:
adding,